scss/index.scss

```diff
@@ -0,0 +1,126 @@
+@import "imports/variables";
+
+@font-face {
+  font-family: Ketchum;
+  src: url('../fonts/Ketchum.otf');
+}
+
+body {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Cantarell', sans-serif;
+}
+
+#scroll {
+  width: 1000px;
+  background-color: $mine-shaft-grey;
+  height: 700px;
+  margin: 0 auto;
+  overflow: auto;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+h2 {
+  margin-bottom: 0;
+  margin-top: 2%;
+}
+
+.poke-container {
+  width: 100%;
+  height: 150px;
+  outline-style: solid;
```

Comment:
using outlines is causing borders to double up also outline-offset -2px is unusual way to achieve a border.  Consider 
border-left:1px solid #222222
border-bottom:1px solid #222222
border-right:1px solid #222222

Reply:
done in 
3aa992d with border: 1px solid #222222;